config/webpack/webpack.config.test.js

```diff
@@ -42,5 +46,16 @@ module.exports = {
   devServer: {
     port: WDS_PORT,
     noInfo: false
-  }
+  },
+  // https://stackoverflow.com/questions/64475910/replacing-polyfill-for-process-in-webpack-v5-from-v4
```

Comment:
Do the other webpack builds (like our library) need these polyfills too?

Reply:
Looks like yes. We're using `process` in our log util: https://github.com/FormidableLabs/victory/blob/main/packages/victory-core/src/victory-util/log.js